config/config.json

```diff
@@ -146,7 +146,8 @@
         "MaxBurst": 100,
         "MemoryStoreSize": 10000,
         "VaryByRemoteAddr": true,
-        "VaryByHeader": ""
+        "VaryByHeader": "",
+        "GoRoutineHealthThreshold": 1000
```

Comment:
I don't think this should go in `RateLimitSettings` since that's all about API rate limiting. Maybe it should go in `ServiceSettings` like all the random stuff that doesn't have a specific section

Also, the capitalization is incorrect. It should be `GoroutineHealthThreshold`